app/src/main/res/values/strings.xml

```diff
@@ -433,8 +433,9 @@
   <string name="show_hints_and_solution">Show hints and solution</string>
   <string name="hints_andSolution_close_icon_description">Navigate up</string>
   <string name="hints_toolbar_title">Hints</string>
+  <string name="show_hint">Show Hint</string>
+  <string name="show_solution">Show</string>
   <string name="reveal_solution">Reveal Solution</string>
-  <string name="reveal_hint">Reveal Hint</string>
```

Comment:
We should probably keep the old name so that we can keep translations.

Reply:
Restored all `reveal_hint` strings.